src/styles/components/filter-bar.less

```diff
@@ -0,0 +1,120 @@
+.filter-bar {
+
+  display: flex;
+  flex-wrap: wrap-reverse;
+  width: 100%;
+
+  .filter-bar-left, .filter-bar-right {
+
+    display: inline-flex;
+    flex: 0 0 auto;
+    flex-wrap: wrap;
+    max-width: 100%;
+    width: auto;
+
+    .filter-bar-item {
+
+      flex: 0 0 auto;
+      max-width: 100%;
+
+    }
+
+  }
+
+  .filter-bar-right {
+
+    margin-left: auto;
+
+    &:last-child {
+
+      padding-right: 0;
+
+    }
+
+    & > .filter-bar-item {
+
+      &:last-child {
+
+        // This overrides all of the specificity applied by the classes
+        // below.
+        padding-right: 0 !important;
```

Comment:
If I don't miss anything special, it should just work without the `!important`. Besides this, I'd like you to simplify the selector and just use `.filter-bar-item:last-child` instead as there is no need for the extra `& >`.

**Example**
```less
 .filter-bar-right {

    margin-left: auto;

    &:last-child {

      padding-right: 0;

    }

    .filter-bar-item:last-child {

      // This overrides all of the specificity applied by the classes
      // below.
      padding-right: 0;

    }

  }
```